aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -22,22 +22,17 @@ In ASP.NET Core, background tasks can be implemented as *hosted services*. A hos
 
 [View or download sample code](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/fundamentals/host/hosted-services/samples/) ([how to download](xref:index#how-to-download-a-sample))
 
-The sample app is provided in two versions:
-
-* Web Host &ndash; Web Host is useful for hosting web apps. The example code shown in this topic is from Web Host version of the sample. For more information, see the [Web Host](xref:fundamentals/host/web-host) topic.
-* Generic Host &ndash; Generic Host is new in ASP.NET Core 2.1. For more information, see the [Generic Host](xref:fundamentals/host/generic-host) topic.
-
 ## Worker Service template
 
 The ASP.NET Core Worker Service template provides a starting point for writing long running service apps. To use the template as a basis for a hosted services app:
 
 [!INCLUDE[](~/includes/worker-template-instructions.md)]
 
----
-
 ## Package
 
-A package reference to the [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting) package is added implicitly for ASP.NET Core apps.
+An app based on the Worker Service template uses the `Microsoft.NET.Sdk.Worker` SDK and has an explicit package reference to the [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting) package. For example, see the sample app's project file (*BackgroundTasksSample.csproj*).
```

Comment:
Is there somewhere that actually shows the text `Microsoft.NET.Sdk.Worker` in context? E.g. The SDK is specified in the csproj file using `<Project Sdk="Microsoft.NET.Sdk.Worker">`.

Reply:
I thought that telling the reader to look at the sample app would be enough, but I can add that here. Stand-by for next commit.